### PR TITLE
suppress PHG warning saying: PHG_EDGE_SIZE_THRESHOLD is low

### PR DESCRIPTION
**Pull request by vishalkenchan**
Please refer to comments in the code.

### CODE REVIEW

**Comment by vishalkenchan** [comment]
I added test that produces PHG warning if `PHG_EDGE_SIZE_THRESHOLD` is set to default value (0.25).

**Comment by jppelteret** [comment]
Thanks for adding the test!

/run-tests

File: tests/zoltan/tria_zoltan_01.cc [hunk 1]
```diff
@@ -0,0 +1,70 @@
+
```
Comment (davydden) [inline tests/zoltan/tria_zoltan_01.cc]: nitpick: please add the copy-right header

**Comment by masterleinad** [comment]
Actually, the test is passing even without this PR, isn't it? We would not log the warning anyway.

**Comment by vishalkenchan** [comment]
@masterleinad The test wouldn't pass without this PR as the default value of `PHG_EDGE_SIZE_THRESHOLD` will be picked up.

**Comment by masterleinad** [comment]
> @masterleinad The test wouldn't pass without this PR as the default value of PHG_EDGE_SIZE_THRESHOLD will be picked up.

Interesting. I just tried to run the test on a current version and get the expected output even without this PR. What is the output you are getting for this test without modifying `PHG_EDGE_SIZE_THRESHOLD`?

**Comment by vishalkenchan** [comment]
I get the following with the default value:
```
Number of vertices: 3

WARNING: PHG_EDGE_SIZE_THRESHOLD is low (0.250000), resulting in only 0 edges
 remaining.
WARNING: PHG_EDGE_SIZE_THRESHOLD is low (0.250000), resulting in only 0 edges
 remaining.Number of vertices: 9
Number of vertices: 27
```

**Comment by davydden** [comment]
@masterleinad 

> I just tried to run the test on a current version and get the expected output even without this PR.

is it Trilinos 12.12.1? 

This is the line of code which is being triggered for us: https://github.com/trilinos/Trilinos/blob/f0350316239aaf41e8e6b82378612aaedf9cc72c/packages/zoltan/src/phg/phg_build.c#L1774-L1778

**Comment by davydden** [comment]
for the record, here's the explanation of the issue from Trilinos mailing list: https://trilinos.org/pipermail/trilinos-users/2018-January/006119.html

**Comment by masterleinad** [comment]
> is it Trilinos 12.12.1?

No, it's Trilinos-12.10.1. I guess the output only appears for a newer version.

**Comment by davydden** [comment]
> I guess the output only appears for a newer version.

There the same code in `12.10.1` https://github.com/trilinos/Trilinos/blob/trilinos-release-12-10-branch/packages/zoltan/src/phg/phg_build.c#L1774-L1778 , so I am not sure what changed between the two.

Either way, the suggested patch at least fixes this problem for one version of Trilinos.

**Comment by masterleinad** [comment]
Fine with me, it clearly looks reasonable. If there is some output to stdout we will log it in the test output since `initlog` is not called.

**Pull request merged**